src/local_sensitivity/sensitivity_interface.jl

```diff
@@ -22,6 +22,19 @@ function _adjoint_sensitivities(sol,sensealg,alg,g,t=nothing,dg=nothing;
     adj_sol[end][(1:l) .+ length(sol.prob.u0)]'
 end
 
+function _adjoint_sensitivities(sol,sensealg::SteadyStateAdjoint,alg,g,dg=nothing;
+                                   abstol=1e-6,reltol=1e-3,
+                                   kwargs...)
+  cache = SteadyStateAdjointSensitivityFunction(g,sensealg,sol,dg)
+
+  linsolve_tmp = cache.J'\vec(cache.gu')
+  #cache.linsolve(vec(cache.gu), cache.J', vec(cache.linsolve_tmp))
+
+  -linsolve_tmp'*cache.pJ #dg/dp?

```

Comment:
yeah only the discrete form really works here so we should be taking dg/dp. Concrete solve might even be easier since that's the dy value from the backpass.